src/index.ts

```diff
@@ -420,7 +420,12 @@ export default runExtension({
       help: "Run an existing query block and output the results.\n\n1. The reference to the query block\n2. The format to output each result",
       handler:
         ({ proccessBlockText, variables }) =>
-        (arg, format = "(({uid}))") => {
+        (arg, ...args) => {
+          const lastArg = args[args.length - 1];
+          const lastArgIsLimitArg = !Number.isNaN(Number(lastArg));
+          const { format, limit } = lastArgIsLimitArg
+            ? { format: args.slice(0, -1).join(","), limit: Number(lastArg) }
+            : { format: args.join(","), limit: 0 };
```

Comment:
this complicated mess is a by product of ordering mattering in Sbs, which is why the next evolution of SBs in SamePage will have order agnostic parameters.

Basically, we want the format argument to be able to support accepting commas, but we can't put the limit argument before it without breaking existing workflows. So, we only assume it is if the last arg is a number

Reply:
👍